src/components/Widgets/Markdown/serializers/index.js

```diff
@@ -164,10 +165,30 @@ export const remarkToMarkdown = obj => {
    */
   const mdast = obj || u('root', [u('paragraph', [u('text', '')])]);
 
+  const remarkToMarkdownPluginOpts = {
+    commonmark: true,
+    fences: true,
+    pedantic: true,
+    listItemIndent: '1',
+
+    // Settings to emulate the defaults from the Prosemirror editor, not
+    // necessarily optimal. Should eventually be configurable.
+    bullet: '*',
```

Comment:
This doesn't affect how it is read, right? just how we write it?

Reply:
Right, once markdown is transformed to AST, there's no record of which character was used for things like bullets. This sets which character to use when stringifying back to markdown, which happens when switching from visual editor to markdown mode, and also when saving the entry.